java/connector-node/run.sh

```diff
@@ -0,0 +1,7 @@
+RISINGWAVE_ROOT=$(git rev-parse --show-toplevel)
```

Comment:
We already have a script to start the connector node service. `java/connector-node/assembly/scripts/start-service.sh`